cms/app/assets/tailwind/application.css

```
@@ -0,0 +1,278 @@
+   ======================================== */
+.markdown-body {
+  --fgColor-default: #1f2328;
+  --fgColor-muted: #59636e;
+  --fgColor-accent: #0969da;
+  --bgColor-default: #ffffff;
+  --bgColor-muted: #f6f8fa;
+  --bgColor-neutral-muted: #818b981f;
+  --bgColor-attention-muted: #fff8c5;
+  --borderColor-default: #d1d9e0;
+  --borderColor-muted: #d1d9e0b3;
+  --fontStack-monospace: ui-monospace, SFMono-Regular, "SF Mono", Menlo, Consolas,
```

Stylelint (17.2.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)